### PR TITLE
Adds The Crying Children (PR Rejection Speedrun)

### PR DESCRIPTION
**Pull request by scionalu**
## About The Pull Request

Yes, actually adds crying children as an abno
All code is literally done, idc if it gets rejected, maybe someone else can use it for other abos
Tldr:
- Adds Crying Children (Including gift, ego, and realized with ability)
- Adds LoR burn status (goes down by 1/2 instead of 1/3

https://github.com/vlggms/lobotomy-corp13/assets/80381079/afa67d1d-b161-4c97-b949-bc88601af8af

Explanation:
1. LoR Burn
Currently only crying children and the ego deals with burn. The ego specifically benefits from this burn
Burn stacks up to 50, which deals true damage every 5 seconds and gets halved right after
If burn ticked twice without any new stack being added, it will disappear
Both of the ego and realized have resistance to burn (I was reluctant to make a new var in suit.dm for this, so it's checked on the status effect instead). When having more burn than your burn res, you'll get a burning overlay. If it gets to 50 stacks, you'll get the human torch overlay instead.
how to apply burn:
Mob/Living/L -> L.apply_lc_burn(number of stack)

## Spoilers
2. The Crying Children
When 2 people died (salv/yuna), the counter goes down (oh how original)
The funny part is here. You can increase the counter on attachment, but the rate is horribly low
If you get a neutral you have a chance to get cursed, if you get bad you're guaranteed to get cursed and the counter lowers again. It has 2 qli counter, so if you tried attach on 1 and get bad, it just breaches and curses you lmao
The curse is randomly chosen from mute, deaf, and the worst one being blind
This curse goes away after 3 minutes and don't worry, I already added a check if you're already mute/deaf/blind. So no, you can't cheese him to get rid of your neg quirk, go simp WN or something.
Before anyone asks, the code is O-01-430 because Philip's birthday is April 30th

Actual breach info
The breach consists of 3 Phases. In every phase, he has a counter that increases by time. That counter reduces everytime you deal 20% damage to him, kill the children, or change phases. When the counter is full, he will burn everyone on the same Z level, dealing massive red and full burn stack (basically extra 75 true damage).
1st Phase
3 Minute timer
Kind of a joke, WAW health and resist with low-ish ALEPH damage
Attack consists of Salvador melee, Yuna AoE, and piercing laser Wound of Sorrow

2nd Phase or what I'd like to call hide-and-seek
5 Minute timer
Splits into 3 child, and gives all 3 curses to everyone in the same Z level. This curse doesn't go away by time like the work curse. You have to find and kill the corresponding children to get rid of the curses. Manager have a big role to find the unseeing child first, which makes you blind (manager can still put beacons anyway). This is especially torture because I made them able to spawn in hallways and main room at random

3rd and Final Phase
1 Minute timer (oh god oh fuck)
Deals twice the burn stack, has actual ALEPH health and resist, and have 1.5 new attacks. (Anti gang violence)
Wounds of Sorrow now shoots at everyone in sight instead of one person
Combusting Courage big ass AoE with big damage and burn (it has 1.5 sec buildup, your fault for not getting out)

3. Gear
Combusting Courage and Scorching Desperation
- Armor
CC and SD has burn resist, reducing incoming burn damage by 15 and 25 respectively. The SD realized ability gives you 50 burn stacks for 30 justice and turns off your burn safety (stack doesn't get halved each tick) for 15 secs. This is actually not really bad because 50 stacks is a massive buff for the weapon that doesn't go away for 15 secs. You also only get 50 damage from this because of the burn res.
The stat is 8/4/6/6 and 8/4/6/8
This is because I feel like he's always been weak minded. After joining the reverb he gets a new resolve, but he's still Philip

- Weapon
An actual obtainable Aleph shield that doesn't come from the well.
Reductions are 3/3/3/3 on normal, and 3/5/4/3 when wearing crying armor, realized as well. Compliments the missing stat (150 max for hybrid)
A shield that actually has a counter mechanic, dealing damage and burning everyone nearby
Every attack and block gives you and your enemy burn stack. You trade high damage for having burn applied to yourself
If you activate the weapon again during block, you will do a dash attack, spending all burn. (The damage is really low on lower stacks, but really high on 40/50) it's actually not that easy to get 40/50, you need time because it gets halved every 5 sec. Or block a lot I guess
Wearing the corresponding armors also increase the burn stack you gain and deal. Having more burn also gives extra burn stack to the enemy (Doesn't help you gain more though)

## Notes

Holy shit the sprites are terrible

## Why It's Good For The Game

Idk

Thus end my Schizo rambling, I am sick, I need to sleep, Why did I make this

## Changelog
:cl:
add: Crying Children, Burn Status, ego, realized, gift
soundadd: multiple crying children stuff, burn_block and burn_sword
imageadd: a lot of icons, status, realized ability, ego icons, and abno icons
/:cl:

### CODE REVIEW

**Comment by Coxswain-Navigator** [comment]
I was going to flat out reject this at first, but I'll allow it with a condition - this will need to be fitted to a certain game mode that allows for any abnormality. The pull request adding this feature is https://github.com/vlggms/lobotomy-corp13/pull/955, and I would put this on hold until that is finished. At the least, I would look at optimizing this as some of the numbers look a little bit off to me.

**Comment by Kitsunemitsu** [comment]
This is being looked at by maintainers, no action is being taken at this time.

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]
Overall strong and cool but we might want to use it for something else. Also make it a bit larger when combined I think.

File: code/game/objects/items/ego_weapons/aleph.dm
Comment (scionalu) [reply code/game/objects/items/ego_weapons/aleph.dm:1066]: It's a 3 sec block, I didn't want it to be too strong as a hybrid shield when getting hit with fast attacks/multiple mobs

File: code/modules/mob/living/simple_animal/abnormality/aleph/crying_children.dm
Comment (scionalu) [reply code/modules/mob/living/simple_animal/abnormality/aleph/crying_children.dm:121]: Yeah, was recommended to do this for smile, completely forgot about Agib tbh
Do return FALSE on death work? I was confused reading apostle/scaredy code

File: code/modules/mob/living/simple_animal/abnormality/aleph/crying_children.dm
Comment (scionalu) [reply code/modules/mob/living/simple_animal/abnormality/aleph/crying_children.dm:106]: Ah, which one is that?

File: code/modules/mob/living/simple_animal/abnormality/aleph/crying_children.dm
Comment (scionalu) [reply code/modules/mob/living/simple_animal/abnormality/aleph/crying_children.dm]: No wonder it took so long when testing lmao
Made it +2 charge instead, the checks and reductions are all even numbers anyway

File: code/modules/mob/living/simple_animal/abnormality/aleph/crying_children.dm
Comment (scionalu) [reply code/modules/mob/living/simple_animal/abnormality/aleph/crying_children.dm]: Weird, it didn't come up when trying to build
changed charge cooldown and time to desperation cooldown and time instead

File: code/datums/status_effects/debuffs.dm
Comment (scionalu) [reply code/datums/status_effects/debuffs.dm]: I completely forgot to put a reset, done

File: code/modules/mob/living/simple_animal/abnormality/aleph/crying_children.dm
Comment (scionalu) [reply code/modules/mob/living/simple_animal/abnormality/aleph/crying_children.dm]: done

**Comment by scionalu** [comment]
Again, I can just scrap the abno code and put it in tegumob instead
Maybe just make it drop the ego when it dies

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
I would like to keep this, change it's can-spawn variable to false, making it admin only, and we can 100% use it later.

That's my final statement on the status of this PR.

File: code/modules/mob/living/simple_animal/abnormality/aleph/crying_children.dm
Comment (scionalu) [reply code/modules/mob/living/simple_animal/abnormality/aleph/crying_children.dm]: changed so it'll call death instead and actually gibbed if it's on final phase

**Comment by scionalu** [comment]
EVERYTHING BROKE
RESTARTING

**Pull request closed without merging**
